dockerfiles/pytorch-dev/install_rocm_from_tarballs.sh

```diff
@@ -0,0 +1,103 @@
+#!/bin/bash
+set -euo pipefail
+
+# Ensure wget is installed
+if ! command -v wget >/dev/null 2>&1; then
+  echo "[INFO] wget not found. Installing..."
+  if command -v apt-get >/dev/null 2>&1; then
+    apt-get update
+    apt-get install -y wget
+  else
+    echo "[ERROR] wget installation not supported on this OS. Please install it manually."
+    exit 1
+  fi
+fi
+
+# Configuration
+RELEASE_TAG="${RELEASE_TAG:-nightly-release}"
+ROCM_VERSION_DATE="${ROCM_VERSION_DATE:-$(date -d '3 days ago' +'%Y%m%d')}"
+ROCM_VERSION_PREFIX="6.4.0rc"
+INSTALL_PREFIX="${INSTALL_PREFIX:-/therock/build/dist/rocm}"
+OUTPUT_ARTIFACTS_DIR="${OUTPUT_ARTIFACTS_DIR:-/rocm-tarballs}"
+GITHUB_RELEASE_BASE_URL="https://github.com/ROCm/TheRock/releases/download"
+
+# Parse AMDGPU targets from input
+if [[ $# -ge 1 ]]; then
+  AMDGPU_TARGETS="$1"
+else
+  AMDGPU_TARGETS="gfx942 gfx1100 gfx1201"
```

Comment:
Maybe the default should be restricted to just one arch?

Reply:
Thinking like only `gfx942`  -  updated